fineract-accounting/src/main/java/org/apache/fineract/accounting/glaccount/api/GLAccountsApiResourceSwagger.java

```diff
@@ -203,23 +203,23 @@ private PutGLAccountsResponsechangesSwagger() {}
         public PutGLAccountsResponsechangesSwagger changes;
     }
 
-    @Schema(description = "DeleteGLAccountsRequest")
-    public static final class DeleteGLAccountsRequest {
+    @Schema(description = "DeleteGLAccountsResponse")
+    public static final class DeleteGLAccountsResponse {
 
-        private DeleteGLAccountsRequest() {
+        private DeleteGLAccountsResponse() {
 
         }
 
-        private static final class DeleteGLAccountsRequestchangesSwagger {
+        private static final class DeleteGLAccountsResponseChangesSwagger {
```

Comment:
Do we need this?

Reply:
Questionable, i am not sure where we are at the moment with removing all swagger definitions or not...

Reply:
Sorry, i meant do we need this empty class in the response spec?

Reply:
Ohh... i havent even checked what it is doing...just fixed the name as it is not request :D